portfolio/src/main/java/com/google/sps/servlets/ChartServlet.Java

```diff
@@ -0,0 +1,41 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.servlets;
+
+import com.google.gson.Gson;
+import java.io.BufferedReader;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/* Servlet to read chart data from data store and return it in doGet */
+@WebServlet("/chart-data") public class ChartServlet extends HttpServlet {
+  @Override public void doGet(HttpServletRequest request,
+                              HttpServletResponse response) throws IOException {
+    List<DataPair> dataPairs = new ArrayList<DataPair>();
+
+    Gson gson = new Gson();
+
+    // Send the JSON as the response
+    response.setContentType("application/json");
+    response.getWriter().println(gson.toJson(dataPairs));
```

Comment:
nit: you can convert to json using 1 line
>new Gson().toJson(dataPairs)

Reply:
That makes sense.